tools/clang/lib/Headers/hlsl/vk/spirv.h

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2024 Google LLC
+//
+// This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
```

Comment:
@llvm-beanz Let me know if this works with your change in https://github.com/microsoft/DirectXShaderCompiler/pull/6872